modules/nw-dpu-configuring-operator.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * networking/networking_operators/configuring-dpu-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-dpu-configuring-operator_{context}"]
+= Configuring the DPU Operator 
+
+Create a `DpuOperatorConfig` custom resource (CR) on both the host cluster and on each of the DPU clusters. The DPU Operator in each cluster is activated after this CR is created.
+
+. Create a file named `dpu-operator-host-config.yaml` using the following YAML:
++
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: DpuOperatorConfig
+metadata:
+ name: dpu-operator-config <1>
+spec:
+ mode: host <2>
```

Comment:
```suggestion
 mode: host <2>
 # ...
```

Unless it's a complete YAML def.

Reply:
This is the complete YAML identifying node as host.